interfaces/interfaces_helpers.go

```diff
@@ -0,0 +1,82 @@
+package interfaces
+
+import (
+	"bufio"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/TudorHulban/GoLayouter/objects"
+)
+
+const _pathInput = "../test_cases/folder_c6"
+
+// to evoid imported cycle
+func IRWritterReadFile(filePath string) ([]string, error) {
+	fileHandler, errOp := os.Open(filePath)
+	if errOp != nil {
+		return nil, errOp
+	}
+
+	var errClo error
+	defer func() {
+		errClo = fileHandler.Close()
+	}()
+
+	var res []string
+
+	scanner := bufio.NewScanner(fileHandler)
+	for scanner.Scan() {
+		res = append(res, scanner.Text())
+	}
+
+	return res, errClo
+}
+
+func TestConvertToIWritter(t *testing.T) {
```

Comment:
looks like a test function?  
should be in a test file!!

Reply:
addressed in fa38b125828f208631c482c74923ffb3fbe4f1da
or 

https://github.com/TudorHulban/GoLayouter/blob/fa38b125828f208631c482c74923ffb3fbe4f1da/domain/interfaces/interfaces_test.go#L25